libs/blocks/graybox-promote/components/graybox-promote-lit.js

```diff
@@ -168,25 +167,23 @@ class GrayboxPromote extends LitElement {
             </p>
             <button @click="${() => this.getSpTokenTask.run()}">Login</button>
           `;
-        } else {
-          return html` <p>Are you sure you want to promote the content for ${experienceName}?</p>
+        }
+        return html` <p>Are you sure you want to promote the content for ${experienceName}?</p>
             <button
-              @click="${() =>
-                this.promoteTask.run({
-                  experienceName,
-                  promoteUrl,
-                  promoteDraftsOnly,
-                  promoteIgnorePaths,
-                  spData,
-                  repo,
-                  ref,
-                  owner,
-                  url,
-                })}"
+              @click="${() => this.promoteTask.run({
+    experienceName,
+    promoteUrl,
+    promoteDraftsOnly,
+    promoteIgnorePaths,
+    spData,
+    repo,
+    ref,
+    owner,
+    url,
+  })}"
```

Comment:
Indentation seems off
![Screenshot 2024-06-26 at 11 39 11](https://github.com/adobecom/milo/assets/11267498/d09912f9-5ce6-4365-84ce-07df5efca01d)

Reply:
Not sure why, but the eslint formatter wants it this way. 🤔